src/main/resources/db/migration/V1.0__modify_dummy_data_table.sql

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Prefer not to have files of commented out code - if we don’t want this data insertion to be run automatically, you can keep it in a file in a separate folder with a comment of when it should be ran.